tests/bzlmod/semver_test.bzl

```diff
@@ -77,8 +77,23 @@ def _semver_test_impl(ctx):
 
 semver_test = unittest.make(_semver_test_impl)
 
+def _humanize_test_impl(ctx):
+    env = unittest.begin(ctx)
+
+    # remove everything after and including the frist + as it is not preserved in a comparable
+    RESTORABLE_VERSIONS = [version.split("+")[0] for version in _SORTED_TEST_VERSIONS]
+
+    for version in RESTORABLE_VERSIONS:
+        humanized = humanize_comparable_version(semver.to_comparable(version, relaxed = RESTORABLE_VERSIONS))
```

Comment:
Why this value for `relaxed`?

Reply:
The existing test data provided by `_SORTED_TEST_VERSIONS` includes versions such as `0.1-a`, which trigger the following faillure:
 
> Semantic version strings must have exactly three dot-separated components, got: 0.1-a

I figure it's worth testing those as well, so I used relaxed for those versions. I can gladly take a different approach, please advise.

Reply:
After poking around a little more, I realized the original versions were actually present and all this code could be safely removed.